src/mixer/playermanager.cpp

```diff
@@ -450,7 +450,7 @@ void PlayerManager::addDeckInner() {
 }
 
 void PlayerManager::loadSamplers() {
-    m_pSamplerBank->loadSamplerBankFromPath(getDefaultSamplerPath(m_pConfig));
+    m_pSamplerBank->loadSamplerBankFromPath(getDefaultSamplerPath(m_pConfig), true);
```

Comment:
You may work around the anonymous bool
```suggestion
    bool initializing = true; 
    m_pSamplerBank->loadSamplerBankFromPath(getDefaultSamplerPath(m_pConfig), initializing);
```